docs/readmes/lte/build_install_magma_pkg_in_agw.md

```diff
@@ -74,3 +78,80 @@ hide_title: true
     ```bash
     apt show magma
     ```
+
+## Steps with Bazel
+
+This is not supported for 1.8, but is possible on the development branch and will replace the documentation above in future versions.
+
+1. **Clone or Update  your local repository**.
+    In any machine (i.e. your computer) clone the magma repo.
+
+    ```bash
+    git clone https://github.com/magma/magma.git
+    ```
+
+2. **Install prerequisites**.
+    Make sure you have installed all the tools specified in the prerequisites <https://magma.github.io/magma/docs/basics/prerequisites#prerequisites>
+
+3. **Build and create deb package**.
+    To build an AGW package spin up a vagrant machine and then build and create a deb package.
+
+    From `$MAGMA_ROOT/lte/gateway` on your host machine run:
+
+    ```bash
+    vagrant up magma
+    vagrant ssh magma
+    ```
+
+    In the VM from `$MAGMA_ROOT` run:
+
+    ```bash
+    bazel run //lte/gateway/release:release_build --config=production
+    ```
+
+    To create a package for development or testing, run
+
+    ```bash
+    bazel run //lte/gateway/release:release_build
+    ```
+
+    Omitting the `--config=production` flag will compile all C++ services with `Debug` compiler flags and enable ASAN. This is recommended for testing only as it will impact performance. In contrast, the production package has C++ services built with `RelWithDebInfo` compiler flags.
+
+4. **Locate the packages**.
+    Once the above command finished you can find the packages inside the VM:
+
+    ```bash
+    cd /tmp/packages
+    ```
+
+    There should be two packages named `magma_1.9.XXX` and `magma-sctpd_1.9.XXX` (for v1.9 versions).
+
+5. **Download the package**.
```

Comment:
I would probably not include this in the documentation. It's not very specific to the build process. Whoever follows this will need a basic understanding of how to copy files between machines anyway (for example in the next section, "upload to agw"). And this is somewhat opinionated, some people would just use `scp` without Vagrant plugin, some would rather copy it to $MAGMA_ROOT etc.